tls/s2n_server_key_exchange.c

```diff
@@ -247,7 +284,26 @@ int s2n_kem_server_key_send(struct s2n_connection *conn, struct s2n_blob *data_t
     GUARD(s2n_kem_generate_keypair(&conn->secure.s2n_kem_keys));
 
     data_to_sign->size = sizeof(kem_extension_size) + sizeof(kem_public_key_size) +  public_key->size;
+    return 0;
+}
+
+int s2n_hybrid_server_key_send(struct s2n_connection *conn, struct s2n_blob *total_data_to_sign)
+{
+    const struct s2n_kex *kex = conn->secure.cipher_suite->key_exchange_alg;
+    const struct s2n_kex *hybrid_kex_0 = kex->hybrid[0];
+    const struct s2n_kex *hybrid_kex_1 = kex->hybrid[1];
+
+    /* Keep a copy to the start of the whole structure for the signature check */
+    total_data_to_sign->data = s2n_stuffer_raw_read(&conn->handshake.io, 0);
+    notnull_check(total_data_to_sign->data);
+
+    struct s2n_blob data_to_verify_0 = {0};
+    GUARD(s2n_kex_server_key_send(hybrid_kex_0, conn, &data_to_verify_0));
+
+    struct s2n_blob data_to_verify_1 = {0};
+    GUARD(s2n_kex_server_key_send(hybrid_kex_1, conn, &data_to_verify_1));
 
+    total_data_to_sign->size = data_to_verify_0.size + data_to_verify_1.size;
```

Comment:
Missing blank line before return to match style elsewhere.

Reply:
I actually think I was too generous with the whitespace in places. Updated everything to match no blank lines before the return.